example/src/App.js

```diff
@@ -1354,6 +1317,7 @@ export default class App extends Component {
         activeNotificationData={this.state.activeNotificationContent}
         defaultTab={this.state.defaultTab}
         autoChartAggregations={this.state.autoChartAggregations}
+        hasQueryTopics={true}
```

Comment:
Since this will be exposed to the public, we may want to stay consistent with the previous naming. Something like:

enableQueryQuickStartTopics

We also need to make sure to update the public docs when this is released. https://chata.readme.io/docs/react-data-messenger#props